src/plugins/nest.js

```diff
@@ -0,0 +1,294 @@
+'use strict'
+
+function createWrapCreate (tracer, config) {
+  return function wrapCreate (create) {
+    return function createWithTrace () {
```

Comment:
A comment or updated name would be helpful since you aren't tracing the create function, but instead its returned handler.  I think this applies in other cases below as well.